api/pkg/apps/seeder/static.go

```diff
@@ -491,6 +491,74 @@ var (
 					},
 				},
 			},
+			{
```

Comment:
These changes should not be made to the `UGICLAIndividualIntake` case template, but should be added as attributes.

An example of the global attributes can be seen here: https://github.com/nrc-no/core/blob/2f03a416ed2f2bc90837e7fa748773a74bdf518d/api/pkg/apps/seeder/static.go#L76

Further, this ticket needs to introduce a mechanism to handle country-attributes. The existing attributes are all global (apply to all countries). The attributes in this ticket are specific to the Ugandan context.

The user should be presented with the combination of the global attributes and the country attributes (inferred from the country/countries they are associated with via the team memberships they have).